src/bot/random_word.py

```diff
@@ -0,0 +1,44 @@
+"""This file contains the functions to be called by the bot that are used to get a random word from the dictionary and ask the user to translate it
+"""
+import random
+import os
+import pandas as pd
+from src.repository.vocabulary import get_all_words, get_or_create_user_id_in_DB
+
+
+def send_random_word(bot, message):
+    """
+    Function that sends a random word from the dictionary to the user to translate
+
+    args:
+    bot: the bot object to send the message
+    message: the message object from the user
+    """
+    global user_known_words
```

Comment:
Why do you define this as a `global` variable here? That doesn't look right, as you want it on a per-user basis. 🤔